### PR TITLE
Ryan new folder

### PR DESCRIPTION
**Pull request by RyanAbraham**
I made a new folder and moved html.php into it. :100: :100: :100: :100: :100: :100: :100: :100: :100: :100: :100: :100: :100: :100: :100:

### CODE REVIEW

File: newfolder/html.php [hunk 1]
```diff
@@ -34,4 +34,4 @@
 <b><font color="<?php echo $color;?>"> Welcome to the website</font></b>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (JacksonSchoe) [inline newfolder/html.php]: This is crap

File: newfolder/html.php [hunk 2]
```diff
@@ -34,4 +34,4 @@
 <b><font color="<?php echo $color;?>"> Welcome to the website</font></b>
```
Comment (JacksonSchoe) [inline newfolder/html.php]: This is better

File: newfolder/html.php (hunk 1 above)
Comment (JacksonSchoe) [inline newfolder/html.php]: WTF is this

Comment (RyanAbraham) [inline newfolder/html.php]: Who made this anyway

**Pull request merged**